packages/lesswrong/server/permissions/accessFilters.ts

```diff
@@ -42,6 +42,14 @@ const clientIdCheckAccess: CheckAccessFunction<'ClientIds'> = async (currentUser
   return userIsAdmin(currentUser)
 }
 
+const commentCheckAccess: CheckAccessFunction<'Comments'> = async (currentUser, document, context): Promise<boolean> => {
+  if (!document.draft) {
```

Comment:
👍 

Double-checking that revisions behave correctly for draft comments?  I think `revisionCheckAccess` is fine in terms of handling draft revisions, but do draft comments get draft revisions made for them, like draft posts?